gapis/server/server.go

```diff
@@ -152,23 +168,47 @@ func (s *server) GetDevicesForReplay(ctx context.Context, p *path.Capture) ([]*p
 
 	all := bind.GetRegistry(ctx).Devices()
 	filtered := make([]bind.Device, 0, len(all))
-nextDevice:
+nextAndroidDevice:
```

Comment:
The main benefit is that it gives us a nice deterministic order to what devices will be returned if we ever run into the case where we have more than one MemoryLayout that match (Host/Android).

I.e. we will always have Android devices first, then Host. So you know that Android will be the first in the list in this case.